pinot-core/src/test/java/com/linkedin/pinot/queries/PregeneratedHllTest.java

```diff
@@ -1,153 +0,0 @@
-/**
- * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.linkedin.pinot.queries;
-
-import com.linkedin.pinot.common.query.gen.AvroQueryGenerator;
-import com.linkedin.pinot.core.indexsegment.utils.AvroUtils;
-import com.linkedin.pinot.core.startree.hll.HllConfig;
-import com.linkedin.pinot.core.startree.hll.HllUtil;
-import com.linkedin.pinot.core.startree.hll.SegmentWithHllIndexCreateHelper;
-import com.linkedin.pinot.util.TestUtils;
-import java.io.File;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import org.apache.avro.Schema;
-import org.apache.avro.file.DataFileStream;
-import org.apache.avro.file.DataFileWriter;
-import org.apache.avro.generic.GenericData;
-import org.apache.avro.generic.GenericDatumWriter;
-import org.apache.avro.generic.GenericRecord;
-import org.apache.commons.io.FileUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.testng.Assert;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.Test;
-
-
-public class PregeneratedHllTest {
```

Comment:
This test needs to exist to test segment generation for pregenerated hll column. It's okay to use different (non-default) value to limit segment size